AsyncMethodNameFixer/AsyncMethodNameFixer.Test/AsyncMethodNameFixerUnitTests.cs

```diff
@@ -332,7 +350,37 @@ public IAwaitable MyMethod(string input)
             }
         }
     }";
-            ExpectMissingAsync(test, "MyMethod", 11, 31);
+            ExpectMissing(test, "MyMethod", 11, 31);
+        }
+
+        [TestMethod]
+        public void Should_Give_Warning_And_Fix_If_Task_Returning_Property_Name_Ends_With_Async()
```

Comment:
@priyanshu92 this gives the clearest indication of what I believe should be the correct behaviour.